.github/workflows/regen_dependabot_conf.yml

```diff
@@ -37,9 +37,16 @@ jobs:
           wget https://raw.githubusercontent.com/nephio-project/test-infra/main/tools/dependabot/generate_dependabot.py -O /tmp/generate_dependabot.py
           python3 /tmp/generate_dependabot.py
 
+      - uses: tibdex/github-app-token@v1
```

Comment:
Where can I find more information about the tibdex/github-app-token and its usage examples? Thank

Reply:
Here it is: https://github.com/tibdex/github-app-token

Reply:
@radoslawc Thanks, 
Where would one find the documentation of supported fields such as secrets.HELPER_APP_ID and secrets.HELPER_APP_KEY

Reply:
Those are GitHub repository secrets https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions 
They are needed to use authentication app to generate ephemeral tokens instead of using PAT
https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens